src/HexCalc.java

```diff
@@ -0,0 +1,312 @@
+		System.out.println("8: Decimal to Hexadecimal");
+		System.out.println("9: Hexadecimal to Decimal");
+		System.out.println("0: Exit");
+	}
+	public static String add(String one,String two)
+	{
+		/*
+		 * Takes two Hexadecimal String as input and return first plus second in Hexadecimal notation
+		 * make use of helper functions toDecimal to convert it into base 10
+		 * then perform the add operation by helper function add 
+		 */
+		return toHexaDecimala(add(toDecimal(one),toDecimal(two)));
+	}
+	public static int add(int a,int b)
+	{
+		/*
+		 * Takes two integer as input and return first plus second Integral value
+		 */
+		return a+b;
+	}
+	public static String subtract(String one,String two)
+	{
+		/*
+		 * Takes two Hexadecimal String as input
+		 * Return Absolute value in Hexadecimal notation
+		 * make use of helper functions toDecimal to convert it into base 10
+		 * then perform the subtract operation by helper function subtract 
+		 */
+		return toHexaDecimala(Math.abs(subtract(toDecimal(one),toDecimal(two))));
+	}
+	public static int subtract(int a,int b)
+	{
+		/*
+		 * Takes two integer as input and return first minus(-) second Integral value
+		 */
+		return (a-b);
+	}
+	public static String multiply(String one,String two)
+	{
+		/*
+		 * Takes two Hexadecimal String as input and return first multiply by second in Hexadecimal notation
+		 * make use of helper functions toDecimal to convert it into base 10
+		 * then perform the multiply operation by helper function multiply 
+		 */
+		return toHexaDecimala(multiply(toDecimal(one),toDecimal(two)));
+	}
+	public static int multiply(int a,int b)
+	{
+		/*
+		 * Takes two integer as input and return first multiply second Integral value
+		 */
+		return a*b;
+	}
+	public static String divide(String one,String two)
+	{
+		/*
+		 * Takes two Hexadecimal String as input and return first divide by second in Hexadecimal notation
+		 * make use of helper functions toDecimal to convert it into base 10
+		 * then perform the divide operation by helper function divide 
+		 */
+		return toHexaDecimala(divide(toDecimal(one),toDecimal(two)));
+	}
+	public static int divide(int a,int b)
+	{
+		/*
+		 * Takes two integer as input and return first divide by second Integral value
+		 * Divide by zero exception is checked and 0 is produced instead of Infinity
+		 */
+		try
+		{
+			return a/b;
+		}
+		catch(ArithmeticException e)
+		{
+			return 0;
+		}
+	}
+	public static int toDecimal(String str)
+	{
+		/*
+		 * This function takes input as hexadecimal value in String Format and return the Integer with base 10
+		 * This function make use of hexa_key to decode Hexadecimal notation.
+		 */
+		int result=0;
+		str=str.toUpperCase();
+		for(int i=0;i<str.length();i++)
+		{
+			char c=str.charAt(i);
+			int d=hex_key.indexOf(c);
+			result=(result*16)+d;
+		}
+		return result;
+	}
+	public static String toHexaDecimala(int num)
```

Comment:
Incorrect Method Name, it should be `toHexaDecimal`.
Recommended way is it should be self identifiable and descriptive. For example you could use intToHexaDecimal instead of toHexaDecimal.

Reply:
Changed to : intToHexaDecimal()